sdk/firstload/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "highlight.run",
-	"version": "5.4.0",
+	"version": "5.4.1",
```

Comment:
could you also please update `CHANGELOG.md` in this directory describing the change?